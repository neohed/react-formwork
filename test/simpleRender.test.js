import React from 'react';
import SimpleForm from './forms/simpleForm';
import renderer from 'react-test-renderer';

test('Test a simple render', () => {
    const component = renderer.create(
        <SimpleForm />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
