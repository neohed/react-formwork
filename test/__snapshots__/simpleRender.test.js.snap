// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test a simple render 1`] = `
<div>
  <form
    name="simple-form"
  >
    <div>
      <label
        htmlFor="name"
      >
        Name
      </label>
      <input
        name="name"
        onBlur={[Function]}
        onChange={[Function]}
        type="text"
        value=""
      />
      
    </div>
    <div>
      <label
        htmlFor="email"
      >
        Email
      </label>
      <input
        name="email"
        onBlur={[Function]}
        onChange={[Function]}
        type="text"
        value=""
      />
      
    </div>
    <button
      disabled={false}
      type="submit"
    >
      Submit
    </button>
  </form>
</div>
`;
